packages/docs/components/splash/MockJSClient.tsx

```diff
@@ -0,0 +1,26 @@
+import { Diagram, NodeDescription } from '@data-story/core';
+import { WorkspacesApi } from '@data-story/ui';
+
+export class JSClient {
+  private digram: Diagram;
+
+  constructor(digram: Diagram) {
+    this.digram = digram;
+  }
+
+  workspacesApi: WorkspacesApi = {
+    getNodeDescriptions: async({ path }) => {
+      return [] as NodeDescription[]
+    },
+    getTree: async({ path }) => {
+      return Promise.resolve({
+        path: '/',
+        type: 'file',
+        content: this.digram,
+        id: 'root',
```

Comment:
On **id**: Could it be discountinued in favor of path? Having path constitute the id since it will be unique?
On **name**, can we have it be the /path/{THIS-IS-THE-NAME}, ie the last part of the path?
This way we could keep things slim, and utilize id/name already being encoded in the tree. However, it might be a hassle retreiving it..

Reply:
I suggest adding `id` and `name` attributes to our data structure, primarily because I've observed that many default functionalities in `react-arborist` rely on these two attributes （https://github.com/brimdata/react-arborist/blob/main/modules/react-arborist/src/components/default-node.tsx). Without them, we'd need to transform attributes within the explorer component.

Reply:
Perhaps, we could also consider customizing the attribute names via [data-with-different-property-names](https://github.com/brimdata/react-arborist?tab=readme-ov-file#data-with-different-property-names) 😂.

Reply:
I see, it seems reasonable to mirror arborist in that way. Lets keep them👍